### DISCUSSION TITLE
claims

### DISCUSSION THREAD

**Discussion by TDMidkwhatisxd** [GriefPrevention/GriefPrevention#1102] [Q&A] [answered]
Is there a way to not allow players to claim near another player claim? For example to set a radius?

**Comment by bigpresh** [comment, marked as answer]
No - the general idea is you claim the area you want to protect, so if you want more of a buffer around your build, make your claim larger.

This is a duplicate of #124 and #505 and #936 - see the discussions on those but the overall consensus was that it is unnecessary and would cause confusion, so won't be happening in GP core; it's possible someone could create a GP addon for it, though.

Closing as a duplicate of the above.
